Tests/MathLib/TestCVode.cpp

```diff
@@ -159,7 +159,7 @@ TEST(MathLibCVodeTest, Exponential)
 	ode_solver->setFunction(f, nullptr);
 	ode_solver->setTolerance(1e-8, 1e-6);
 
-	ode_solver->setIC(t0, {{y0}});
+	ode_solver->setIC(t0, {&y0});
```

Comment:
It would be nice if all of the y's would be in the same domain (like Eigen::Matrix). On the other hand creating of the Eigen::Matrices is not possible with initializer lists, if I'm not mistaken.
Switching back to std::array allows usage of initializer lists, but is not consistent with the space where the y's do live.